### PR TITLE
Adding Redshift Schema Dump & Restore Functionality

### PR DESCRIPTION
**Pull request by davekrupinski**

### CODE REVIEW

**Comment by davekrupinski** [comment]
@dweinand Updated to support view schema dump.

File: lib/postshift.rb [hunk 1]
```diff
@@ -1,5 +1,24 @@
 require 'postshift/version'
+require 'postshift/railtie' if defined?(Rails)
+require 'postshift/schema'
 require 'active_record/connection_adapters/redshift_adapter'
 
 module Postshift
+  @@adapter = nil
```
Comment (dweinand) [inline lib/postshift.rb]: Let's use `thread_cattr_accessor` here instead.

**Comment by davekrupinski** [comment]
@dweinand ready for another look

**Comment by davekrupinski** [comment]
@dweinand my senior moment has passed, ready for another look.

**Comment by geneshuman** [comment]
Sigh, I would like to unsubscribe from your newsletter :P

Im not sure why I keep getting valimail github notifications.  I am not
watching any of your repos, and I've thoroughly checked my settings, an
there is no direct connection to valimail in my github settings.  And I've
never even heard of that repo.

So is this something on you're guys' end?

Regardless, I hope you are all doing well :)

**Comment by apanzerj** [comment]
Hi Gene!

**Pull request merged**